### DISCUSSION TITLE
AWS EKS Istio Ambient pod connectivity on different nodes

### DISCUSSION THREAD

**Discussion by tomahkvt** [istio/istio#49056] [Q&A]
I have an AWS EKS Cluster with different networks for Nodes and pods. I have configurated Istio Ambient Mesh at this cluster using this instruction https://istio.io/latest/docs/ops/ambient/getting-started/. I don't have connectivity between pods on different nodes.

**Comment by howardjohn** [comment]
The older versions of ambient do not work on many platforms, probably your EKS setup was one of them. The new builds from `master` (merged last week) include some new work that should make it work. See https://github.com/istio/istio/wiki/Dev-Builds for info on pulling latest builds
